tests/test_nmf_mfcc.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import unittest
+import numpy as np
+import nussl
+import os
+
+class NMFMFCCUnitTests(unittest.TestCase):
+
+    # Update this if the benchmark file changes and rerun freeze_duet_values() (below)
+    path_to_benchmark_file = os.path.join('..', 'Input', 'piano_and_synth_arp_chord_mono.wav')
+
+    def load_benchmarks(self):
+        benchmark_dict = {}
+        directory = 'nmf_mfcc_reference/nmf_mfcc_benchmarks'
+        for filename in os.listdir(directory):
+            key = os.path.splitext(filename)[0]
+            file_path = os.path.join('nmf_mfcc_reference', 'nmf_mfcc_benchmarks', filename)
+            value = np.load(file_path)
+            benchmark_dict[key] = value
+        return benchmark_dict
+
+    def test_mfcc_initialization(self):
```

Comment:
I was confused about some of the comments in here because I didn't understand what this was supposed to test. Add a comment here with a sentence or two about what this function is testing. Probably not a bad idea to document all of the functions in here.

Reply:
Also, the name of this function is misleading. `mfcc` could be shorthand for `nmf_mfcc`.